gulp_tasks/webpack.js

```diff
@@ -7,20 +7,20 @@ const webpackDistConf = require('../conf/webpack-dist.conf');
 const gulpConf = require('../conf/gulp.conf');
 const browsersync = require('browser-sync');
 
-gulp.task('webpack:dev', done => {
+gulp.task('webpack:dev', (done) => {
```

Comment:
Ideally, functions should explicitly `return` at some point. In ES6, single `return` lambdas can be placead on the same line. I would recommend either:

```
gulp.task('webpack:dev', (done) => {
  return webpackWrapper(false, webpackConf, done);
};
```

or:

```
gulp.task('webpack:dev', (done) => webpackWrapper(false, webpackConf, done));
```

you can also omit the `parens` on single line lambdas but I prefer not to:

```
gulp.task('webpack:dev', done => webpackWrapper(false, webpackConf, done));
```